brainkeeper/src/assets/people.json

```diff
@@ -0,0 +1,36 @@
+{ 
```

Comment:
Are we keeping it so that there are always person’s in here to start using the app with, or is this file temporary?

Reply:
This is a temporary solution to have some persons available at start during development.
But this could also be used to store (maybe German) celebrities to be available for the user without importing persons.